unitycatalog-ai/core/pyproject.toml

```diff
@@ -55,11 +55,13 @@ include = [
   "src/*"
 ]
 exclude = [
-  "src/ucai/test_utils/*"
+  "src/unitycatalog/ai/test_utils/*"
 ]
 
 [tool.hatch.build.targets.wheel]
-packages = ["src/ucai"]
+sources = ["src"]
+only-include = ["src/unitycatalog/ai/core"]
+packages = ["src/unitycatalog"]
```

Comment:
This is the "new" (as of Python 3.4) way of constructing namespace packages according to https://packaging.python.org/en/latest/guides/packaging-namespace-packages/ where the omission of __init__.py at the shared namespace(s) level is the recommended approach. 
We are intentionally not doing the legacy approach (https://packaging.python.org/en/latest/guides/packaging-namespace-packages/#legacy-namespace-packages) that involves specifying the `pkg_resources` package from setuptools due to its marked deprecation and the higher level of complexity involved / higher chances of accidentally releasing a plugin module that will effectively break all other packages in the shared namespace.